runtime/plugin/matchit.vim

```diff
@@ -1,8 +1,10 @@
 "  matchit.vim: (global plugin) Extended "%" matching
-"  Last Change: Fri Jul 29 01:20 AM 2016 EST
+"  Last Change: 2016 Aug 21
 "  Maintainer:  Benji Fisher PhD   <benji@member.AMS.org>
 "  Version:     1.13.2, for Vim 6.3+
 "		Fix from Tommy Allen included.
```

Comment:
Thanks for the effort (not just on this issue) @chrisbra!  I thought I originally emailed Benji before I commented on that old PR, but I can't find the email.  It seems he only replied when Bram cc'd him.  Maybe he's not subscribed to the list email notifications?  Still strange that he skipped the PR dealing with the jumps, though.